### PR TITLE
version: update name

### PR DESCRIPTION
**Pull request by selsta**

### CODE REVIEW

File: src/version.cpp.in [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #define DEF_MONERO_VERSION_TAG "@VERSIONTAG@"
 #define DEF_MONERO_VERSION "0.15.0.0"
-#define DEF_MONERO_RELEASE_NAME "Carbon Chamaeleon"
+#define DEF_MONERO_RELEASE_NAME "Nitrogen Nebula"
```
Comment (moneromooo-monero) [inline src/version.cpp.in:3]: Still at 0.15.0.0

Comment (selsta) [reply src/version.cpp.in:3]: Yep, until #6514 is merged

Comment (moneromooo-monero) [reply src/version.cpp.in:3]: OK, I see the same was already merged to the branch now.

**Comment by dEBRUYNE-1** [comment]
@selsta - Needs a rebase.

**Pull request merged**